runtime/src/parachain.rs

```diff
@@ -28,10 +28,12 @@ macro_rules! runtime_parachain {
 			transaction_version: 2,
 		};
 
-		impl cumulus_parachain_upgrade::Config for Runtime {
+		impl cumulus_parachain_system::Config for Runtime {
 			type Event = Event;
 			type OnValidationData = ();
 			type SelfParaId = ParachainInfo;
+			type DownwardMessageHandlers = ();
```

Comment:
@4meta5 Looks like there is at least something you'll be able to play with in this version.